ci/build-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+execution_log_postfix=${1:-}${2:-}
+
+export LC_ALL=en_US.UTF-8
+
+ARTIFACT_DIRS="${BUILD_ARTIFACTSTAGINGDIRECTORY:-$PWD}"
+mkdir -p "${ARTIFACT_DIRS}/logs"
+
+if [ "${1:-}" = "_m1" ]; then
+    bazel="arch -arm64 bazel"
+else
+    bazel=bazel
+fi
+
+$bazel build \
+       //compiler/damlc/tests/platform-independence.dar \
```

Comment:
Why this target, specifically?

Reply:
I've been looking through the script and that seems to be the only one we need to build that is not already built by the script that needs it.

Making that sentence a little bit more concrete, if you look at [copy-unix-release-artifacts.sh](https://github.com/digital-asset/daml/blob/6e5763b40f88ae1af7a91bcf51785a307cd291a5/ci/copy-unix-release-artifacts.sh) for example, you'll see it explicitly calls the relevant `bazel` commands it needs.